__tests__/V4AWSClusterManagement.js

```diff
@@ -79,7 +79,7 @@ afterAll(() => {
 // and hence the warning because we are transforming an array into an array
 it('renders all the v4 AWS cluster data correctly without nodes ready', async () => {
   const div = document.createElement('div');
-  const { getByText, getAllByText, getByTestId } = renderRouteWithStore(
+  const { getByText, getAllByText, getByTestId, debug } = renderRouteWithStore(
```

Comment:
debug not used here.
```suggestion
  const { getByText, getAllByText, getByTestId } = renderRouteWithStore(
```